common/path_distance_calculator/src/path_distance_calculator.cpp

```diff
@@ -46,10 +46,12 @@ PathDistanceCalculator::PathDistanceCalculator(const rclcpp::NodeOptions & optio
     }
     if (path->points.empty()) {
       RCLCPP_WARN_THROTTLE(get_logger(), *get_clock(), 5000, "path empty");
+      return;
     }
 
     if (path->points.size() == 1) {
       RCLCPP_WARN_THROTTLE(get_logger(), *get_clock(), 5000, "too short path");
+      return;
     }
```

Comment:
These can be merged into `path->points.size() <= 1`.

Reply:
Thansk.
addressed in [85a3ab8](https://github.com/autowarefoundation/autoware.universe/pull/2892/commits/85a3ab85d26b14a01e852d75ac065690e8474a00)